### PR TITLE
feat: Removed product tour from home page

### LINKED ISSUE #24058: Remove product Tour from Home page in popup view
The Product Tour was added when we made the new changes in the header to update the users about it. This needs to be removed in order to make the onboarding smoother
[Private Zenhub Video](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBeEJtQkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--5c741a53937880e5ca24cc649d3cb30ff362f126/Screen%20Recording%202024-04-16%20at%209.05.34%E2%80%AFPM.mov)

### PR DESCRIPTION
**Pull request by NidhiKJha**
This PR is to remove product tour from home page

## **Related issues**

Fixes: #24058 

## **Manual testing steps**

1. Install fresh extension dev build
2. switch to popup view
3. Check there is no product tour on homepage

### **Before**

https://github.com/MetaMask/metamask-extension/assets/39872794/4fbcec7d-5644-4d04-bda6-2ffaedeb1b97

### **After**

https://github.com/MetaMask/metamask-extension/assets/39872794/008c4707-1ecc-4686-882d-2daa9e3a3bdf

## **Pre-merge author checklist**

- [ ] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've completed the PR template to the best of my ability
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

**Comment by darkwing** [comment]
Doesn't this require some cleanup within global state?

**Review by jonybur — APPROVED** [review]

**Comment by jonybur** [comment]
Getting: `Too long with no output (exceeded 10m0s): context deadline exceeded` on the CI, other than that all good

File: ui/components/multichain/app-header/app-header.js [hunk 1]
```diff
@@ -45,32 +45,23 @@ import {
 import {
   getCurrentChainId,
   getCurrentNetwork,
-  getOnboardedInThisUISession,
   getOriginOfCurrentTab,
-  getShowProductTour,
```
Comment (jonybur) [inline ui/components/multichain/app-header/app-header.js:50]: Please remove from global store as it is not used anymore

**Review by jonybur — CHANGES_REQUESTED** [review]

**Comment by NidhiKJha** [comment]
@darkwing @jonybur I am thinking to do the global state update in a separate PR since it will need a migration as well. With this one merged, we could remove the product tour from the new Multichain build. Or should I do the migration in this PR?

**Comment by jonybur** [comment]
@NidhiKJha What sort of migration would be necessary?

**Review by jonybur — APPROVED** [review]

**Review by sahar-fehri — APPROVED** [review]

**Pull request merged**